src/storage/mongodb/CarStorage.ts

```diff
@@ -686,10 +688,11 @@ export default class CarStorage {
             $cond: {
               if: { $gt: ['$carCatalog.image', null] }, then: {
                 $concat: [
-                  `${Utils.buildRestServerURL()}/client/util/CarCatalogImage?ID=`,
+                  `${Utils.buildRestServerURL()}/v1/util/car-catalogs/`,
                   '$carCatalog.id',
-                  '&LastChangedOn=',
-                  { $toString: '$carCatalog.lastChangedOn' }
+                  {
```

Comment:
@Ninjeneer Missing '/image' in URL here.
@mahdibenromdhane found the issue.

Reply:
Fixed in #3151